aten/src/ATen/native/quantized/cpu/qnnpack/src/deconv-run.cc

```diff
@@ -19,7 +83,242 @@ enum pytorch_qnnp_status qnnpackDeConv(
     const uint8_t output_zero_point,
     const bool transpose,
     uint8_t* output,
-    pthreadpool_t threadpool);
+    pthreadpool_t threadpool) {
+
+  if (batch_size == 0) {
+    // Doesn't matter what's going on, if no batches, return
+    return pytorch_qnnp_status_success;
+  }
+
+  // Check all invalid parameters
+  const size_t kernel_width = deconv_p.kernel_dims[0];
+  const size_t kernel_height = deconv_p.kernel_dims[1];
+  if (kernel_width == 0 || kernel_height == 0) {
```

Comment:
BTW, why are we doing these checks here? Can this not to be done in deconv_param_t constructor. Conv op does that so we should probably maintain the convention. Even better if we can just use conv_params_t if possible.

Reply:
Yeah I think @z-a-f  mentioned this as a todo in the previous PR. I think we can merge them

Reply:
This check is done in the params. I can remove it from here